docs/2.8.0/docs/daml-finance/packages/interfaces/changelogs/daml-finance-interface-instrument-generic.rst

```diff
@@ -4,14 +4,28 @@
 Daml.Finance.Interface.Instrument.Generic - Changelog
 #####################################################
 
+Version 3.0.0
+*************
+
+- Update of SDK version and dependencies.
+
+- The `InstrumentKey` is extended by the `HoldingStandard` field.
+
+- The `Remove` choice was removed from the `Factory` (it is newly part of the `Base` instrument
+  interface).
+
+- Renamed the `F` type synonym to `I`.
+
 Version 2.0.0
 *************
 
 - Update of SDK version and dependencies
 
-- The `Create` choice on the instrument factory returns the corresponding interface (rather than the base instrument interface)
+- The `Create` choice on the instrument factory returns the corresponding interface (rather than the
```

Comment:
Another section without terminal punctuation -- I'm going to stop marking them now but maybe do an overall check if you do want to ensure consistency. Totally up to you.

Reply:
will do, thanks